routeros/resource_interface_ethernet.go

```diff
@@ -94,10 +94,10 @@ func ResourceInterfaceEthernet() *schema.Resource {
 			DiffSuppressFunc: AlwaysPresentNotUserProvided,
 		},
 		"cable_settings": {
-			Type:        schema.TypeString,
-			Optional:    true,
-			Description: `Changes the cable length setting (only applicable to NS DP83815/6 cards)`,
-			ValidateFunc: validation.StringInSlice([]string{"default", "short", "standard"}, false),
+			Type:             schema.TypeString,
```

Comment:
My editor auto-runs go fmt all the time, let me know if I need to revert this spacing.

Reply:
No need for that, thank you!